src/mpid/ch4/src/ch4r_callbacks.c

```diff
@@ -381,7 +381,7 @@ int MPIDIG_send_target_msg_cb(int handler_id, void *am_hdr, void **data, size_t
         /* MPIDI_CS_ENTER(); */
         while (TRUE) {
             rreq = MPIDIG_dequeue_posted(hdr->src_rank, hdr->tag, hdr->context_id,
-                                         &MPIDIG_COMM(root_comm, posted_list));
+                                         is_local, &MPIDIG_COMM(root_comm, posted_list));
```

Comment:
There's a typo in the commit message for 6176e7b83801a12d5eb7ea9f6177fc14a4850f34:

```diff
- This is necessary when both shmmod and netmod are using activemessages
+ This is necessary when both shmmod and netmod are using active messages
and an anysource recv pair are posted in the same posted_list. Without
- it, a netmod send will gets matched to a shmmod recv, resulting in
+ it, a netmod send will get matched to a shmmod recv, resulting in
errors.
```